user-guide/Advanced_Functionality/DataMiner_Systems/Adding_a_DMA/Adding_a_DMA_to_a_DMS_running_STaaS.md

```diff
@@ -0,0 +1,33 @@
+---
+uid: Adding_a_DMA_to_a_DMS_running_STaaS
+---
+
+# Adding a DataMiner Agent to a DMS running STaaS
+
+Follow these steps below to connect a DMA to an existing DMS running STaaS:
+
+1. **Ensure the DMA is Online and Empty**: The DMA needs to be online and empty before you start the process.
+
+1. **Disconnect from Cloud**: The DMA cannot be cloud-connected during this process. Ensure that it is disconnected from any cloud services by following this procedure: [Permanently disconnecting from dataminer.services](https://docs.dataminer.services/user-guide/Cloud_Platform/AboutCloudPlatform/Disconnecting_from_dataminer.services.html#permanently-disconnecting-from-dataminerservices)
+
+1. **Start the Agent without the STaaS Flag**: Start the DMA, but make sure the active flag is set to **false** in the *DB.xml* file. It should look like this:
```

Comment:
Does this mean the DMA should previously have been stopped? The first step suggests to me that it has to be running, so this isn't entirely clear.

Reply:
it should be restarted after changing the flag to false